File: SlickBlog.Web/Models/Post.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SlickBlog.Web.Models
{
    public class Post
    {
        public int Id { get; set; }
        public Blog Blog { get; set; }
        public string Title { get; set; }
        public string ContentText { get; set; }
        public string[] Tags { get; set; }
        public Comment Comments { get; set; }
    }
}

File: Spa.Web/App/viewmodels/post.js
define(function () {
    var Post = function() {
        var self = this;
        self.id = ko.observable();
        self.userId = ko.observable();
        self.title = ko.observable().extend({ required: true });
        self.contentText = ko.observable().extend({ required: true });
        self.tags = ko.observableArray();
        self.comments = ko.observableArray();
        self.blogId = ko.observable();
    };

    return Post;

});